modApiServer/src/org/aion/api/server/rpc3/PersonalRPC.java

```diff
@@ -0,0 +1,46 @@
+package org.aion.api.server.rpc3;
+
+import static org.aion.api.server.rpc3.RPCExceptions.*;
+import static org.aion.api.server.rpc3.types.RPCTypes.*;
+import static org.aion.api.server.rpc3.types.RPCTypesConverter.*;
+
+/******************************************************************************
+*
+* AUTO-GENERATED SOURCE FILE.  DO NOT EDIT MANUALLY -- YOUR CHANGES WILL
+* BE WIPED OUT WHEN THIS FILE GETS RE-GENERATED OR UPDATED.
+*
+*****************************************************************************/
+public abstract class PersonalRPC{
+
+    /**
+     *Allows you to interact with accounts on the aion network and provides a handful of crypto utilities
+     */
+    public String execute(String requestString){
+        try{
+
+        Request request = RequestConverter.decode(requestString);
+        //check that the request can be fulfilled by this class
+        String interfaceName = request.method.split("_")[0];
+        if(interfaceName.equals("personal")){
```

Comment:
check `interfaceName` not equal to null

Reply:
That is not necessary RPCTypes.Requests marks that field as required and throws if it is not found. So the field will always be defined.